components/Pagination/src/Pagination.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import clsx from 'clsx';
+import { PaginationPrevious } from './PaginationPrevious';
+import { PaginationNext } from './PaginationNext';
+import { PaginationList } from './PaginationList';
+import { PaginationLink } from './PaginationLink';
+
+interface Props {
+  index?: number;
+  indexLimit: number;
+  onChange?: (index: number) => number;
+  className?: string;
+}
+
+export const Pagination = ({ index = 0, indexLimit, onChange, className }: Props) => {
+  const [currentIndex, setCurrentIndex] = React.useState(index);
```

Comment:
Niet belangrijk, maar ik zou useState hier gewoon direct importeren en gebruiken zoals we overal doen.